### PR TITLE
Feature/create quick flight reservation

### PR DESCRIPTION
**Pull request by MasterRadule**

### CODE REVIEW

File: tim1/src/main/java/isamrs/tim1/service/ReservationService.java [hunk 1]
```diff
@@ -386,5 +392,56 @@ private void inviteFriendToFlight(String email, FlightReservationDTO flightRes,
...
+		Flight f = flightRepository.findOneByFlightCode(quickDTO.getFlightCode());
+		if (f == null) {
+			return new MessageDTO("Flight does not exist.", ToasterType.ERROR.toString());
+		}
+		QuickFlightReservation qfr = new QuickFlightReservation();
+		double price = 0.0;
+		String[] idx = quickDTO.getSeat().split("_");
+		int row = Integer.parseInt(idx[0]);
+		int column = Integer.parseInt(idx[1]);
+		if (checkIfSeatIsReserved(f, row, column)) {
```
Comment (draganculibrk9) [inline tim1/src/main/java/isamrs/tim1/service/ReservationService.java]: Dates of reservations should also be checked
https://github.com/ISAMRS-TIM1/Projekat/blob/7a8defc8cf1d9e2b16b13cc2b97d776bab899128/tim1/src/main/java/isamrs/tim1/service/ReservationService.java#L273

File: tim1/src/main/java/isamrs/tim1/service/ReservationService.java [hunk 2]
```diff
@@ -386,5 +392,56 @@ private void inviteFriendToFlight(String email, FlightReservationDTO flightRes,
...
+		return new MessageDTO("Quick flight reservation successfully created", ToasterType.SUCCESS.toString());
+	}
 
+	public ArrayList<QuickFlightReservationDTO> getQuickFlightReservations() {
+		AirlineAdmin airlineAdmin = (AirlineAdmin) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
+		Airline a = airlineAdmin.getAirline();
+		ArrayList<QuickFlightReservationDTO> quickRes = new ArrayList<QuickFlightReservationDTO>();
+		Set<FlightReservation> fRes = a.getReservations();
+		for(FlightReservation fr : fRes) {
+			if (fr instanceof QuickFlightReservation) {
```
Comment (aleksaleks123) [inline tim1/src/main/java/isamrs/tim1/service/ReservationService.java]: Add this check in _if condition_: fr.getUser()==null

File: tim1/src/main/java/isamrs/tim1/service/ReservationService.java (hunk 1 above)
Comment (MasterRadule) [reply tim1/src/main/java/isamrs/tim1/service/ReservationService.java]: One seat can not be reserved for different dates.

File: tim1/src/main/java/isamrs/tim1/service/ReservationService.java (hunk 2 above)
Comment (MasterRadule) [reply tim1/src/main/java/isamrs/tim1/service/ReservationService.java]: Corrected

**Pull request merged**